### ISSUE TITLE
Wrong include -  #include "core_watchdog.c"

### ISSUE THREAD

**Issue by kgoveas** [aethaniel/ExperimentalCore-sam#55]
Hi,

First of all great job with the Arduino core for the Atmel boards :+1: 

I got a multiple definition error due to the include in the core_main.cpp being 
``` c 
#include "core_watchdog.c" 
```
It should be  
``` c
#include "core_watchdog.h" 
```

Would recommend shortening the included headers in the core_main.cpp to
``` c
#include "Arduino.h"
#if defined(USBCON)
//#include ""
#endif
```

Best,
Kevin

**Comment by aethaniel** [comment]
Hi Kevin,
Sorry, I missed your issue (no mail notifications).

Fixed in commit f1b7fc5

Thanks!

**Comment by kgoveas** [comment]
Is there a plan to have this core available in the latest Arduino IDE? or is this project currently archived?
